app/src/main/res/layout-sw600dp-port/resume_lesson_fragment.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.resumelesson.ResumeLessonViewModel" />
+  </data>
+
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/component_color_shared_screen_primary_background_color"
+    android:overScrollMode="never"
+    android:scrollbars="none">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:paddingBottom="24dp">
+
+      <org.oppia.android.app.customview.LessonThumbnailImageView
+        android:id="@+id/resume_lesson_chapter_thumbnail_image_view"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:adjustViewBounds="true"
+        app:entityId="@{viewModel.chapterSummary.explorationId}"
+        app:entityType="@{viewModel.entityType}"
+        app:layout_constraintDimensionRatio="16:9"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:lessonThumbnail="@{viewModel.chapterSummary.chapterThumbnail}" />
+
+      <TextView
+        android:id="@+id/resume_lesson_chapter_title_text_view"
+        style="@style/TextViewStart"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="70dp"
+        android:layout_marginTop="32dp"
+        android:layout_marginEnd="70dp"
+        android:layout_marginBottom="16dp"
```

Comment:
Could you please ensure that all dimensions are not hardcoded, but included from the `dimens.xml` file? Also, we use a factor of 4 for all dimens, please see [wiki](https://github.com/oppia/oppia-android/wiki/Working-on-UI#paddingmarginview-dimensions). The If no dimens exist fo r this screen please feel free to create new ones, and observe the file structure.

Reply:
Also, shall I use `marginHorizontal` instead of `marginStart` & `marginEnd`?

Reply:
It's best to use start and end

Reply:
Also, I am curious why factor of 4 is preferred here?

Reply:
Okay, thanks!